### PR TITLE
Eng | Add strong name validation to package validator

### PR DESCRIPTION
**Pull request by JRahnama**

### CODE REVIEW

File: eng/pipelines/common/templates/jobs/validate-signed-package-job.yml [hunk 1]
```diff
@@ -208,7 +237,7 @@ jobs:
         }
       }
     displayName: 'Verify all DLLs unzipped match "expected" hierarchy'
-
+
```
Comment (DavoudEshtehari) [inline eng/pipelines/common/templates/jobs/validate-signed-package-job.yml:240]:
```suggestion

```

File: eng/pipelines/common/templates/jobs/validate-signed-package-job.yml [hunk 2]
```diff
@@ -106,6 +106,35 @@ jobs:
       }
     displayName: 'Verify nuget signature'
 
+  - powershell: |
```
Comment (DavoudEshtehari) [inline eng/pipelines/common/templates/jobs/validate-signed-package-job.yml:109]: Shouldn't this be skipped on other runs except the official pipeline with the official template?

File: eng/pipelines/common/templates/jobs/validate-signed-package-job.yml (hunk 1 above)
Comment (JRahnama) [reply eng/pipelines/common/templates/jobs/validate-signed-package-job.yml:247]: there is an extra line in after each task, why should we remove this one?

File: eng/pipelines/common/templates/jobs/validate-signed-package-job.yml (hunk 2 above)
Comment (JRahnama) [reply eng/pipelines/common/templates/jobs/validate-signed-package-job.yml:109]: added

**Review by DavoudEshtehari — APPROVED** [review]

**Review by arellegue — APPROVED** [review]

**Pull request merged**